scripts/extend_index_yaml.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script for extending index.yaml.
+In Python 2, the index.yaml autoamtically extended with # AUTOGENERATED,
+but this is no longer true in Python 3.
+
+This script extracts new kind from
+../cloud_datastore_emulator_cache/WEB-INF/index.yaml
+and appends it into index.yaml"""
+
+from __future__ import annotations
+
+import os
+import yaml
+
+
+INDEX_YAML_PATH = os.path.join(os.getcwd(), 'index.yaml')
+WEB_INF_INDEX_YAML_PATH = os.path.join(
+    os.getcwd(), '..', 'cloud_datastore_emulator_cache',
+    'WEB-INF', 'index.yaml'
+)
+
+
+def open_yaml_as_dict(yaml_file):
+    """Opens yaml file and returns it as a dictionary.
+
+    Args:
+        yaml_file: str. Path to yaml file.
+
+    Returns:
+        dict. Dictionary of yaml file.
+    """
+    data = {}
+    with open(yaml_file, 'r', encoding='utf-8') as fp:
+        docs = yaml.safe_load_all(fp)
+        for doc in docs:
+            for key, value in doc.items():
+                data[key] = value
+    return data
```

Comment:
Don't we have a function like this in _utils.py_ or _python_utils_?

Reply:
Removed this one.